cmd/cmd.go

```diff
@@ -40,18 +40,24 @@ func NewPackCommand(logger ConfigurableLogger) (*cobra.Command, error) {
 		return nil, err
 	}
 
+	var forceColor bool // Add a variable to store the forceColor flag
+
 	rootCmd := &cobra.Command{
 		Use:   "pack",
 		Short: "CLI for building apps using Cloud Native Buildpacks",
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
 			if fs := cmd.Flags(); fs != nil {
-				if flag, err := fs.GetBool("no-color"); err == nil && flag {
-					color.Disable(flag)
-				}
-
-				_, canDisplayColor := term.IsTerminal(logging.GetWriterForLevel(logger, logging.InfoLevel))
-				if !canDisplayColor {
-					color.Disable(true)
+				if forceColor { // Check if the forceColor flag is set
```

Comment:
I think you are missing to expose to set this value as a flag, I pulled and compiled your branch for example:

running:

```bash
pack build --help 

# I can't see the "--force-color" flag.

Global Flags:
      --no-color     Disable color output
  -q, --quiet        Show less output
      --timestamps   Enable timestamps in output
  -v, --verbose      Show more output
```

You need to add your flag:

```go
rootCmd.PersistentFlags().Bool("force-color", false, "<some description>")
```

And use the `GetBool` function to evaluate if it was set

Reply:
Thank you reviewing!! I will check it out..

Reply:
@jjbustamante  I don't know why pack command not working sarthak@hp-pavilion:~/pack$ make build
=====> Building...
go build -ldflags "-s -w -X 'github.com/buildpacks/pack.Version=0.0.0+git-5bb10e5' -extldflags " -trimpath -o ./out/pack -a ./cmd/pack
sarthak@hp-pavilion:~/pack$ cd out
sarthak@hp-pavilion:~/pack/out$ ls
pack  tests
sarthak@hp-pavilion:~/pack/out$ pack build --help 
Command 'pack' not found, but can be installed with:
sudo apt install liballegro4-dev